### DISCUSSION TITLE
Which branch of DIGIT-DevOps repo to use for prod environment on AWS? Which has updated script and mainted according to versions of k8s.

### DISCUSSION THREAD

**Discussion by mannansiddiqui** [egovernments/Digit-Core#296] [DIGIT Platform]
@Harish-egov Could you please help here.

**Comment by mannansiddiqui** [comment]
@Harish-egov which branch should i use?

**Reply by naresh-egov** [reply]
@mannansiddiqui Please use the DIGIT-Bootcamp branch for latest AWS Script.  
https://github.com/egovernments/DIGIT-DevOps/tree/DIGIT-Bootcamp
